pkg/corerp/api/v20231001preview/container_conversion.go

```diff
@@ -151,6 +156,64 @@ func (src *ContainerResource) ConvertTo() (v1.DataModelInterface, error) {
 	return converted, nil
 }
 
+// toEnvDataModel: Converts from versioned datamodel to base datamodel
+func toEnvDataModel(e map[string]*EnvironmentVariable) (map[string]datamodel.EnvironmentVariable, error) {
```

Comment:
We can rename this to **toEnvironmentVariableDataModel**.

Reply:
Also renamed the fromEnvVarDataModel to be consistent